specification/communication/data-plane/Rooms/preview/2023-03-31-preview/communicationservicesrooms.json

```
@@ -72,6 +72,42 @@
             "$ref": "./examples/Rooms_CreateRoom.json"
           }
         }
+      },
+      "get": {
+        "tags": [
+          "Rooms"
+        ],
+        "summary": "Retrieves all created rooms.",
+        "operationId": "Rooms_GetRooms",
```

According to Azure convention this should be List* instead of Get*
```suggestion
        "operationId": "Rooms_ListRooms",
```

---

We would have to change this to follow the Noun_Verb format so it would be something like: "Rooms_List" or Rooms_Get and Room_Get for the single room. The pipeline complains how the Id is right now. Any guidance on this?

---

Let's follow what the pipeline wants, I think that's `Rooms_List` and `Rooms_Get`